### PR TITLE
Fix LFNG Info resolution and document sections

### PR DESCRIPTION
**Pull request by sandratoh**
- Add no data placeholders
- Remove duplicated section after refactor
- Update fields render logic to match form

### CODE REVIEW

File: alcs-frontend/src/app/features/application/lfng-info/lfng-info.component.html [hunk 1]
```diff
@@ -121,18 +121,24 @@ <h4>Plans & Bylaws: Zoning</h4>
...
           </div>
           <div>
             <div class="subheading2">Minimum Lot Size (hectares)</div>
             <div>{{ applicationReview.zoningMinimumLotSize }}</div>
+            <app-no-data *ngIf="!applicationReview.zoningMinimumLotSize"></app-no-data>
           </div>
           <div>
             <div class="subheading2">Is this proposal consistent with the current zoning designation?</div>
-            <div>{{ applicationReview.isZoningConsistent ? 'Yes' : 'No' }}</div>
+            <div *ngIf="applicationReview.OCPConsistent !== null">
```
Comment (dhaselhan) [inline alcs-frontend/src/app/features/application/lfng-info/lfng-info.component.html:138]: Should this be isZoningConsistent not OCPConsistent?

Comment (sandratoh) [reply alcs-frontend/src/app/features/application/lfng-info/lfng-info.component.html:138]: Yes, you're right. Thanks for the catch!

Comment (sandratoh) [reply alcs-frontend/src/app/features/application/lfng-info/lfng-info.component.html]: Fixed!

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**